app/presenters/menu/section.rb

```diff
@@ -16,6 +16,10 @@ def visible?
       auth
     end
 
+    def subsection?
```

Comment:
Seems like this is never used here, but it is used (and defined again) in #6946 so wondering if it's intended..?